Kentico.Kontent.Delivery/ContentItems/AssetValueConverter.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Kentico.Kontent.Delivery.Abstractions;
+using Kentico.Kontent.Delivery.ContentItems.Elements;
+using Microsoft.Extensions.Options;
+
+namespace Kentico.Kontent.Delivery.ContentItems
+{
+    internal class AssetValueConverter : IPropertyValueConverter<IEnumerable<IAsset>>
```

Comment:
As discussed & agreed on call, please rename this class to `AssetElementValueConverter`.

Reply:
Renamed.